project_event/models/project_task.py

```diff
@@ -216,6 +217,34 @@ def check_numeric(self):
         except Exception as exception:
             raise ValidationError(_('The Input value should be numeric'))
 
+    real_date_start = fields.Datetime(
+        string='Actual Start Time',
+        default=None,
+    )
+    real_date_end = fields.Datetime(
+        string='Actual End Time',
+        default=None,
+    )
+    total_time = fields.Char(
+        string='Actual Total Time',
+        compute='_compute_actual_total_time',
+    )
+
+    @api.one
```

Comment:
You have to add also a constraint that checks if real_date_end > real_date_start